oleggerasimovich/parser.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import os
+
+from dotenv import load_dotenv
+import psycopg2
+
+load_dotenv()
+
+db = os.getenv('DATABASE')
+password = os.getenv('PASSWORD')
+host = os.getenv('HOST')
+user = os.getenv('USER')
+port = os.getenv('PORT')
```

Comment:
It is better to use `os.environ.get()` instead of `os.getenv`.

Reason is there: https://stackoverflow.com/a/55940480